extensions/tika/runtime/src/main/java/io/quarkus/tika/Content.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.tika;
+
+public class Content {
```

Comment:
It was named `TikaContent` but then I thought it is possibly a bit more precise/less verbose to call it just `Content` because it is just a content from the given file (text and metadata). I have a minor preference to keep it this way but will be OK with renaming back to `TikaContent` if you still prefer it